chainercv/links/model/ssd/transforms.py

```diff
@@ -164,8 +164,7 @@ def random_crop_with_bbox_constraints(
         of the value.
 
         * **constraint** (*tuple*): The chosen constraint.
-        * **y_slice** (*slice*): A slice used to crop the input image.\
-            The relation below holds together with :obj:`x_slice`.
+        * **y_slice** (*slice*): A slice used to crop the input image.
```

Comment:
How about "A slice in vertical direction to crop the input image." for `y_slice` and
"A slice in horizontal direction used to crop the input image." for `x_slice`.
The current doc for `x_slice` is wrong.